src/presetsController.ts

```diff
@@ -726,6 +726,20 @@ export class PresetsController {
         return selectedConfigurePreset;
     }
 
+    private async checkBuildPreset(): Promise<preset.BuildPreset | null> {
```

Comment:
This seems more like a "pickBuildPreset"

Reply:
It is named to match `checkConfigurePreset` above this.  `ensure` might be a better name and I think we've used it elsewhere, but I didn't think it was a big enough deal to change.  "pick" is not the best word choice because this function does not pick.  It will give you an option to pick if one is not selected already, but will do nothing if one is already "picked."  I can change it to "ensure" in both functions if you feel strongly about it.